packages/flutter/lib/src/widgets/scroll_activity.dart

```diff
@@ -274,6 +274,12 @@ class ScrollDragController implements Drag {
   static const Duration momentumRetainStationaryDurationThreshold =
       Duration(milliseconds: 20);
 
+  /// The minimum amount of velocity needed to apply the [carriedVelocity] at
+  /// the end of a drag. Expressed as a factor. For example with a [carriedVelocity]
+  /// of 2000, we will need a velocity of at least 1000 to apply the [carriedVelocity]
+  /// as well. Decided by fair eyeballing with the scroll_overlay platform test.
```

Comment:
Describe what happens if it doesn't meet the threshold. Looking at the video, it seems like a new fling velocity is re-evaluated based on the speed of the last swipe?